config/initializers/carrierwave.rb

```diff
@@ -14,14 +15,24 @@
     config.storage = :file
   else
     config.fog_provider = "fog/aws"
-    region = ApplicationConfig["AWS_UPLOAD_REGION"].presence || ApplicationConfig["AWS_DEFAULT_REGION"]
-    config.fog_credentials = {
-      provider: "AWS",
-      aws_access_key_id: ApplicationConfig["AWS_ID"],
-      aws_secret_access_key: ApplicationConfig["AWS_SECRET"],
-      region: region
-    }
-    config.fog_directory = ApplicationConfig["AWS_BUCKET_NAME"]
     config.storage = :fog
+    if ENV["HEROKU_APP_ID"].present?
+      region = ApplicationConfig["AWS_UPLOAD_REGION"].presence || ApplicationConfig["AWS_DEFAULT_REGION"]
+      config.fog_credentials = {
+        provider: "AWS",
+        aws_access_key_id: ApplicationConfig["AWS_ID"],
+        aws_secret_access_key: ApplicationConfig["AWS_SECRET"],
+        region: region
+      }
+      config.fog_directory = ApplicationConfig["AWS_BUCKET_NAME"]
+    else #SYSTEMSNOTE Non-Heroku for our WIP container
```

Comment:
Should we go with the convention of using the Github team name for all of these comments? 
```ruby
# @forem/systems: ....
```

Reply:
Cool that makes sense